src/schedule/zephyr_domain.c

```diff
@@ -36,8 +36,6 @@ LOG_MODULE_DECLARE(ll_schedule, CONFIG_SOF_LOG_LEVEL);
 
 #define ZEPHYR_LL_STACK_SIZE	8192
 
```

Comment:
care to provide more details in the commit message? Why the next_logic is unnecessary now?

Reply:
There are few comments in the code which already explains that this have to be removed in the future and this can be removed because zephyr ll is strictly periodic I think